tests/e2e/tests/build/platform-server.ts

```diff
@@ -58,7 +58,7 @@ export default function () {
     `))
     .then(() => silentExec(normalize('node'), 'index.js'))
     .then(() => expectFileToMatch('dist/index.html',
-      new RegExp('<h2 _ngcontent-c0="">Here are some links to help you start: </h2>')))
+      new RegExp('<h2>Here are some links to help you start: </h2>')))
```

Comment:
I looked at this with @Brocco and as far as I could tell, `_ngcontent-c0=""` wasn't in the output of prerendering for the versions I tested. Prerendering was still happening correctly though.